src/components/Coaching/CoachingDetail/ActivitySummary/ActivitySummary.tsx

```diff
@@ -58,27 +43,47 @@ export const ActivitySummary: React.FC<ActivitySummaryProps> = ({
 }) => {
```

Comment:
Can we rename this component to `LevelOfEffort` so that when a dev is looking at the page, it's easier to tell which components correspond to which parts of the page?